Client/Catalog/frontend/store/configuration/configuration.effects.ts

```diff
@@ -278,18 +285,27 @@ export class ConfigurationEffects {
 		() =>
 			this.actions$.pipe(
 				ofType(onError),
-				withLatestFrom(this.store$.select(selectProduct), this.store$.select(selectLocale).pipe(map((l) => l || 'de-DE'))),
-				map(([{ message }, product, locale]) => {
-					const sectionName = product.sections.find((e) => e.id === message.errorPayload.section)?.name?.[locale];
-
-					const element: Element<any> | undefined = product.elements.find((e) => e.id === message.errorPayload.element);
-					const elementName = element?.name?.[locale];
+				withLatestFrom(
+          this.store$.select(selectLocale).pipe(map((l) => l || 'de_DE')),
+          this.store$.select(selectContentSnippet('aptoRules.defaultErrorMessage'))
+        ),
+				map(([{ message }, locale, snippet]: [{message: MessageBusResponseMessage}, string, ContentSnippet]) => {
+          const defaultMessage: string = snippet.content[locale];
```

Comment:
Please always use "import {translate} from "@apto-base-core/store/translated-value/translated-value.model";" TranslatedValue. Because we have some fallback logic if no translation for current language is found.